spec.html

```diff
@@ -29445,10 +29445,10 @@ <h1>
               1. If _propDesc_ is not *undefined* and _propDesc_.[[Enumerable]] is *true*, then
                 1. Let _descObj_ be ? Get(_props_, _nextKey_).
                 1. Let _desc_ be ? ToPropertyDescriptor(_descObj_).
-                1. Append the pair (a two element List) consisting of _nextKey_ and _desc_ to the end of _descriptors_.
+                1. Append the Record { [[Key]]: _nextKey_, [[Descriptor]]: _desc_ } to _descriptors_.
             1. For each element _pair_ of _descriptors_, do
-              1. Let _P_ be the first element of _pair_.
-              1. Let _desc_ be the second element of _pair_.
+              1. Let _P_ be _pair_.[[Key]].
```

Comment:
Instead of creating aliases, we can just inline these at their use sites now.

Reply:
Done.